tests/Feature/QuizApiControllerTest.php

```diff
@@ -37,4 +37,38 @@ public function test_quiz_api_controller_no_quizzes_exist(): void
                     ->where('data', []);
             });
     }
+
+    public function test_quiz_api_controller_successful_create_quiz(): void
+    {
+        $quizData = [
+            'name' => 'quiz',
+            'description' => 'quiz',
+        ];
+        $response = $this->postJson('/api/quizzes', $quizData);
+        $response->assertStatus(201)
+            ->assertJson(function (AssertableJson $response) {
+                $response->has('message')
+                    ->whereType('message', 'string');
+            });
```

Comment:
Can you do a database assertion as well? I could delete the bit of code that actually saves it and the tests would still pass